src/toffy/normalize.py

```diff
@@ -81,14 +79,14 @@ def write_mph_per_mass(
 
 
 def create_objective_function(obj_func):
-    """Creates a function of specified type to be used for fitting a curve
+    """Creates a function of specified type to be used for fitting a curve.
 
     Args:
         obj_func (str): the desired objective function. Must be either poly_2, ..., poly_5, or log
 
     Returns:
-        function: the function which will be optimized"""
-
+        function: the function which will be optimized
```

Comment:
The return type should be `Callable` in cases where a function returns another function.